sentry_sdk/integrations/django/transactions.py

```diff
@@ -76,6 +76,8 @@ def _simplify(self, pattern):
             result.replace("^", "")
             .replace("$", "")
             .replace("?", "")
+            .replace("\\A", "")
+            .replace("\\Z", "")
```

Comment:
Since this is probably in the hot path for transactions, might be worth thinking about a more efficient way to carry out the replacements in the future, e.g. https://stackoverflow.com/questions/3367809/efficiently-carry-out-multiple-string-replacements-in-python